templates/paloalto_panos_show_interface_management.textfsm

```diff
@@ -0,0 +1,18 @@
+Value Name (\S+)
+Value MACaddress ([a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2})
+Value IPv4 (\S+)
+Value IPv4Netmask (\S+)
+Value IPv4Gateway (\S+)
+Value IPv6 (\S+)
+Value IPv6LinkLocal (\S+)
+Value IPv6Gateway (\S+)
+
+Start
+  ^Name: ${Name}
+  ^\s+Port MAC address ${MACaddress}
+  ^Ip address: ${IPv4}
+  ^Netmask: ${IPv4Netmask}
+  ^Default gateway: ${IPv4Gateway}
+  ^Ipv6 address: ${IPv6}
+  ^Ipv6 link local address: ${IPv6LinkLocal}
+  ^Ipv6 default gateway: ${IPv6Gateway} -> Record
```

Comment:
Please add the following at the end of the file:
```
  ^\s*$$
  ^. -> Error
```

Reply:
Hmm, can you explain why? I haven't seen that in any of the other textfsm templates.

Reply:
^\s*$$
  ^. -> Error

what is the purpose of this?
thanks

Reply:
`^\s*$$` will account for blank lines
`^. -> Error` will error out with any non captured/accounted for output. This helps detect changes in output and points us in the right direction when troubleshooting templates.